### ISSUE TITLE
AUC_ROC for Multilabel

### ISSUE THREAD

**Issue by agemagician** [automl/auto-sklearn#417]
I need to calculate the AUC_ROC for a Multi-label imbalanced class problem.
Which parameter I should use for the average in the "roc_auc_score" ?
Because each one of them is giving me different output, and I need to determine the correct one.

**Comment by mfeurer** [comment]
The answer to this question depends on your problem setup and can probably be found in the scikit-learn documentation [here](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score). If not, the scikit-learn mailing list should be the right place to get an answer to this question.
